pkg/visor/api.go

```diff
@@ -610,6 +611,36 @@ func (v *Visor) SetAppPK(appName string, pk cipher.PubKey) error {
 	return nil
 }
 
+// SetAppDNS implements API.
+func (v *Visor) SetAppDNS(appName string, dnsAddr string) error {
```

Comment:
hmm, we are setting the DNS per the app here? I suppose that is useful but it makes me think that the DNS is not settable for the visor process itself.